### PR TITLE
Decompile Bolting spell anim

### PR DESCRIPTION
**Pull request by Eebit**

### CODE REVIEW

File: src/banim-efxmagic-thunder.c [hunk 1]
```diff
@@ -145,8 +145,8 @@ void NewEfxThunderOBJ(struct Anim *anim)
     proc->timer = 0;
     proc->anim2 = EfxCreateFrontAnim(anim, gUnknown_085F5550, gUnknown_085F4A24, gUnknown_085F5550, gUnknown_085F4A24);
 
-    SpellFx_RegisterObjPal(gUnknown_085F3F40, 0x20);
-    SpellFx_RegisterObjGfx(gUnknown_085F3AA8, 0x1000);
+    SpellFx_RegisterObjPal(Pal_BoltingSprites, 0x20);
```
[the remark this answers is not in the document]
Comment (Eebit) [reply src/banim-efxmagic-thunder.c:148]: Fair point, yeah! I tried to address this for this file in cde9fd1. I borrowed the `PLTT_SIZE` macros from pokeemerald and changed the graphics calculations to be their size in tiles.

I only did it for this file for now. @MokhaLeee - Maybe at some point between the two of us we can plan to do a bigger clean-up PR?

[the remark this answers is not in the document]
Comment (MokhaLeee) [reply src/banim-efxmagic-thunder.c:148]: > Maybe at some point between the two of us we can plan to do a bigger clean-up PR

That makes sense. You may directly fix all functions related. I'm not recently working on banim related function. So this action may not affect on my work.

[the remark this answers is not in the document]
Comment (Eebit) [reply src/banim-efxmagic-thunder.c:148]: Sounds good - I'll take care of it in the next few days :)

[the remark this answers is not in the document]
Comment (RevoSucks) [reply src/banim-efxmagic-thunder.c:148]: Ok, ill consider it out of scope for this particular PR then and merge since I have no other issues.

**Pull request merged**